### PR TITLE
feat: course enrollment listing updates

### PR DESCRIPTION
**Pull request by DawoudSheraz**
### Description
Adds the course title and pacing information on the enrollment listing. Three existing fields, Modified, Modified By, and Reason have been moved, A new button, Show Extra, has been added under actions. Clicking the button opens a new section that has the above-mentioned 3 fields. This relocation has been done in order to avoid getting the table bulky and hard to read.

Screenshot 2021-05-04 at 4 14 38 PM

------------------

Screenshot 2021-05-04 at 4 14 54 PM

---------------------

Screenshot 2021-05-04 at 4 38 27 PM

### CODE REVIEW

File: src/users/enrollments/Enrollments.jsx [hunk 1]
```diff
@@ -47,15 +69,8 @@ export default function Enrollments({
...
-      reason: {
-        value: result.manualEnrollment ? result.manualEnrollment.reason : '',
-      },
-      lastModifiedBy: {
-        value: result.manualEnrollment ? result.manualEnrollment.enrolledBy : '',
-      },
-      lastModified: {
-        displayValue: result.manualEnrollment ? formatDate(result.manualEnrollment.timestamp) : '',
-        value: result.manualEnrollment ? result.manualEnrollment.timestamp : '',
+      pacing: {
```
Comment (azanbinzahid) [inline src/users/enrollments/Enrollments.jsx]: Suggestion: have the key name as `pacingType`

File: src/users/enrollments/Enrollments.jsx [hunk 2]
```diff
@@ -20,7 +21,25 @@ export default function Enrollments({
...
   const [formType, setFormType] = useState(null);
   const [enrollmentToChange, setEnrollmentToChange] = useState(undefined);
+  const [enrollmentExtraData, setEnrollmentExtraData] = useState(undefined);
   const formRef = useRef(null);
+  const extraRef = useRef(null);
+
+  function setupEnrollmentExtraData(enrollment) {
+    const extraData = {
+      courseName: enrollment.courseName,
+      lastModified: enrollment.manualEnrollment ? formatDate(enrollment.manualEnrollment.timeStamp) : 'N/A',
```
Comment (azanbinzahid) [inline src/users/enrollments/Enrollments.jsx:31]: doesn't `formatDate` return N/A by default?

File: src/users/enrollments/Enrollments.jsx [hunk 3]
```diff
@@ -113,13 +144,7 @@ export default function Enrollments({
...
     },
     {
-      label: 'Reason', key: 'reason', columnSortable: true, onSort: () => setSort('reason'), width: 'col-3',
-    },
-    {
-      label: 'Last Modified By', key: 'lastModifiedBy', columnSortable: true, onSort: () => setSort('lastModifiedBy'), width: 'col-3',
-    },
-    {
-      label: 'Last Modified', key: 'lastModified', columnSortable: true, onSort: () => setSort('lastModifiedBy'), width: 'col-3',
+      label: 'Pacing Type', key: 'pacing', columnSortable: true, onSort: () => setSort('pacing'), width: 'col-3',
```
Comment (azanbinzahid) [inline src/users/enrollments/Enrollments.jsx]: this key will change accordingly

**Review by azanbinzahid — APPROVED** [review]
Just a question and suggestion.

File: src/users/enrollments/Enrollments.jsx (hunk 2 above)
Comment (DawoudSheraz) [reply src/users/enrollments/Enrollments.jsx:31]: Yes, but that is when manualEnrollment has a value. This N/A is applicable when no manualEnrollment entry is present.

**Review by Ali-D-Akbar — APPROVED** [review]

**Pull request merged**